apps/mobile/src/services/get-users.ts

```diff
@@ -0,0 +1,4 @@
+export async function getUser() {
+  const result = await fetch('https://api.github.com/users/diego3g')
+  return await result.json()
```

Comment:
O que acha de usar o Axios? será bem melhor quando formos usar o acess token no backend

Reply:
podemos sim, utilizar o axios acho uma ótima ideia. Posso seguir nessa mesma impletamentação e configura-lo tbm se for de acordo para todos

Reply:
também acho uma boa ideia usar o axios.

Reply:
Suporte ao axios já integrado também junto a essa PR